appveyor.yml

```diff
@@ -14,12 +17,14 @@ install:
   - SET OPENSSL_ROOT_DIR=C:\tools
   - SET PATH=C:\tools\bin;%PATH%
 
+  - ps: wget 'https://s3.amazonaws.com/kylo-pl-bucket/curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z' -OutFile "$pwd\curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z"
+  - ps: 7z.exe x "curl-7.42.1-x86_64_mingw-w64_4.8.3_win32_seh.7z" -oC:\tools | FIND /V "ing "
```

Comment:
That's a mistake in Leatherman. This PR does use a source build of Leatherman on Windows, but it's pulling in a required dependency on libcurl even though you're not requesting the Leatherman.curl component. I'll file a ticket to fix it.